### PR TITLE
[CAL-34961] Adds event-type route and view

### PR DESCRIPTION
**Pull request by RSalaam**
# Changes
- added server-side API to fetch single event-type data
- created single event-type component to render single event-type data (will allow editing later)
- added single event-type path via router
- linked each event-type on dashboard to its corresponding `/event-types/{uuid}` page
- removed "hyperlink-look" of event-type name and description on dashboard
- added new method to format a `last_updated` field in single event-type resource

# Story
https://calendlyapp.atlassian.net/browse/CAL-34961

### CODE REVIEW

File: components/eventType.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    const fetchData = async () => {
+        const result = await fetch(
+            `/api/event_types/${uuid}`
+        ).then((res) => res.json());
+
+        setEventType(result.event)
+    }
+
+    console.log(eventType)
```
Comment (tristanoneil) [inline components/eventType.jsx:18]: Probably want to remove this.

File: utils/index.js [hunk 2]
```diff
@@ -26,3 +26,8 @@ exports.formatEventDateTime = (event) => ({
   start_time: formatTime(event.start_time),
   end_time: formatTime(event.end_time),
 });
+
+exports.formatEventTypeDate = (event) => ({
```
Comment (tristanoneil) [inline utils/index.js:30]: I think `event` here should be `eventType`.

**Review by tristanoneil — CHANGES_REQUESTED** [review]
Just two minor things, otherwise looks great!

File: components/eventType.jsx (hunk 1 above)
Comment (RSalaam) [reply components/eventType.jsx:18]: Aw, I thought I removed all console.log's. Sorry about that. I specifically searched for them, but must have missed this one.

File: utils/index.js (hunk 2 above)
Comment (RSalaam) [reply utils/index.js:30]: Yeah, good idea!

**Review by tristanoneil — APPROVED** [review]

**Pull request merged**